lib/src/command/outdated.dart

```diff
@@ -229,12 +235,19 @@ class OutdatedCommand extends PubCommand {
     if (available.isEmpty) {
       return null;
     }
+
     final prereleases = argResults.wasParsed('prereleases')
```

Comment:
Unrelated to this PR, but maybe give this line a comment about how we support two spellings...

Reply:
Done in: https://github.com/dart-lang/pub/pull/2488